tools/system_libs.py

```diff
@@ -1069,7 +1075,7 @@ class Dummy(object):
   # You can provide 1 to include everything, or a comma-separated list with the ones you want
   force = os.environ.get('EMCC_FORCE_STDLIBS')
   if force == '1':
-    force = ','.join(system_libs_map.keys())
+    force = ','.join(key for key, value in system_libs_map.items() if not value.never_force)
```

Comment:
How about `for name, lib in .. ` so its clear we are building a list of library names?

Reply:
Why not use `name` rather than `key` too?   lgtm either way, this is micro nit.